ls1intum/Artemis
# `Quiz exercises`: Fix multiple choice answer distribution layout in statistics

### Checklist
#### General


#### Client

### Motivation and Context


This PR fixes #4505 

### Description

I declared the list item content as flex container in order to align it with the list prefix

### Steps for Testing

Prerequisites:
- 1 Instructor
- A MC quiz with rather long answer options
- Firefox and another browser of your choice (e.g. Chrome, Edge, Safari, etc.)

1. Log in to Artemis
2. Navigate to Course Administration
3. Go to the exercise overview of the associated course and click on the "Statistics" Button for the quiz
4. Change the view via the drop down menu below to the specific quiz question (below "Point distribution" and "Quiz statistic")
5. Inspect the left side of the view and check whether the list items are aligned with their list prefixes
6. Do the same for the other browser and check that the layout is comparable and not distorted as it was the case for Firefox before.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage





Nothing changed
### Screenshots


You can find screenshots about the current situation in the corresponding issue. 
This is how it should look now:
![image](https://user-images.githubusercontent.com/80622272/147858352-e6f1b578-67f3-442b-8dd4-7490c2168feb.png)
